recipes/modern-xorg-stack/build.sh

```diff
@@ -0,0 +1,57 @@
+#! /bin/bash
+
+# Build a bunch of X.org packages at once. The list of packages below should
+# be kept synchronized with make-tarball.sh.
+
+set -e
+
+export PKG_CONFIG_LIBDIR=$PREFIX/lib/pkgconfig:$PREFIX/share/pkgconfig
+
+if [ -n "$OSX_ARCH" ] ; then
+    export MACOSX_DEPLOYMENT_TARGET=10.6
+    sdk=/
+    export CFLAGS="$CFLAGS -isysroot $sdk"
+    export LDFLAGS="$LDFLAGS -Wl,-syslibroot,$sdk"
+fi
+
+# Work around weird packaging bug? (See also how we weirdly need to give
+# xextproto --disable-specs.)
+cp xcb-proto-*/missing xextproto-*/
+
+# Note that `-e` mode doesn't cause us to exit when the inner pipelines fail,
+# even though they exit with error codes.
+(cd util-macros-* && ./configure --prefix=$PREFIX && make install) || exit $?
```

Comment:
They probably do. I was cavalier since I figured these are well-established, fundamental packages, so I think that it's a safe bet that if the build succeeds they're going to work. But it doesn't hurt to do some checks.